packages/fhir-converter/src/templates/cda/ccd.hbs

```diff
@@ -3,7 +3,7 @@
     "resourceType": "Bundle",
     "type": "batch",
     "entry": [
-      {{>Header.hbs}}
+      {{>Header.hbs}},
```

Comment:
OOC - why is the `,` necessary? couldn't have been broken for this long lol

Reply:
Yah it doesnt fuck up the output